book/10-git-internals/sections/objects.asc

```diff
@@ -328,7 +328,7 @@ $ find .git/objects -type f
 
 If you follow all the internal pointers, you get an object graph something like this:
 
-.All the objects in your Git directory.
+.All the reachable objects in your Git directory.
 image::images/data-model-3.png[All the objects in your Git directory.]
```

Comment:
Can you update the square-bracket text as well? I forget which is which, but the text after the `.` and the text inside the `[`square brackets`]` are both important. One is the figure caption, the other is alt-text when rendered as HTML.